### ISSUE TITLE
[Bug] Topic group

### ISSUE THREAD

**Issue by Tornamic** [telegramdesktop/tdesktop#28570]
### Steps to reproduce

1. Create/find a group chat with topic support
2. Right-click on the marked area on the screenshot
![image](https://github.com/user-attachments/assets/6e3bb20d-6901-4a48-bc87-4459aacbd176)
3. Enjoy the crash!

### Expected behaviour

nothing

### Actual behaviour

crash

### Operating system

Windows 10 x64 22H2

### Version of Telegram Desktop

5.6.3 x64

### Installation source

Static binary from official website

### Logs

```shell
[2024.10.24 14:39:22] Launched version: 5006003, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2024.10.24 14:39:22] Executable dir: C:/Users/Admin/AppData/Roaming/Telegram Desktop/, name: Telegram.exe
[2024.10.24 14:39:22] Initial working dir: C:/Users/Admin/AppData/Roaming/Telegram Desktop/
[2024.10.24 14:39:22] Working dir: C:/Users/Admin/AppData/Roaming/Telegram Desktop/
[2024.10.24 14:39:22] Command line: C:\Users\Admin\AppData\Roaming\Telegram Desktop\Telegram.exe
[2024.10.24 14:39:22] Executable path before check: C:/Users/Admin/AppData/Roaming/Telegram Desktop/Telegram.exe
[2024.10.24 14:39:22] Logs started
[2024.10.24 14:39:22] AppUserModelID: Telegram.TelegramDesktop.b242bd1bb4a25207b7836e6099bf4764
[2024.10.24 14:39:22] Using DirectX compiler 'C:\Users\Admin\AppData\Roaming\Telegram Desktop\modules\x64\d3d\d3dcompiler_47.dll'.
[2024.10.24 14:39:22] Connecting local socket to Global\c0fb219f4852a06252428afb4010aa92-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2024.10.24 14:39:22] This is the only instance of Telegram, starting server and app...
[2024.10.24 14:39:22] Moved logging from 'C:/Users/Admin/AppData/Roaming/Telegram Desktop/log_start0.txt' to 'C:/Users/Admin/AppData/Roaming/Telegram Desktop/log.txt'!
[2024.10.24 14:39:22] Opened 'C:/Users/Admin/AppData/Roaming/Telegram Desktop/tdata/working' for reading, the previous Telegram Desktop launch was not finished properly :( Crash log size: 491
[2024.10.24 14:39:22] Update Info: MTP is unavailable.
[2024.10.24 14:39:22] App Error: _basePath is empty in writeSettings()
[2024.10.24 14:39:22] Global devicePixelRatio: 1
[2024.10.24 14:39:22] QT_DPI_ADJUSTMENT_POLICY: AdjustDpi
[2024.10.24 14:39:22] Primary screen DPI: 96, Base: 96.
[2024.10.24 14:39:22] Computed screen scale: 100
[2024.10.24 14:39:22] DevicePixelRatio: 1
[2024.10.24 14:39:22] ScreenScale: 100
[2024.10.24 14:39:22] Media Info: Type Playback, default: {0.0.0.00000000}.{fc9085dc-8225-4ac5-8579-dfbfdfa33ce6}, list: { "Headphones (High Definition Audio Device)" <{0.0.0.00000000}.{15f0d464-2a06-4a37-9903-7b1b5fb10321}>, "Digital Audio (HDMI) (High Definition Audio Device)" <{0.0.0.00000000}.{733c9f7a-e353-4d03-8928-f28880bc2931}>, "Digital Audio (S/PDIF) (High Definition Audio Device)" <{0.0.0.00000000}.{956e4100-7429-4493-a860-4ed20bc55db0}>, "Цифровой выход (AMD High Definition Audio Device)" <{0.0.0.00000000}.{cdf0f3e3-e08c-46bd-8629-851c84c9a2a1}>, "Внутренний разъем  AUX (High Definition Audio Device)" <{0.0.0.00000000}.{e0e4d7f5-aade-48b3-8e5e-d521c5412c27}>, "Внутренний разъем  AUX (High Definition Audio Device)" <{0.0.0.00000000}.{ed16a72b-35d8-4e45-b8d2-4844559150a4}>, "Динамики (High Definition Audio Device)" <{0.0.0.00000000}.{fc9085dc-8225-4ac5-8579-dfbfdfa33ce6}> }, full list refresh: false
[2024.10.24 14:39:22] Media Info: Type Capture, default: {0.0.1.00000000}.{2f225804-6861-4147-8785-5b8de1272edf}, list: { "Microphone (USB PnP Audio Device)" <{0.0.1.00000000}.{2f225804-6861-4147-8785-5b8de1272edf}>, "Внутренний разъем  AUX (High Definition Audio Device)" <{0.0.1.00000000}.{338182b5-1dbc-4915-83de-c933aee233fe}>, "Microphone (High Definition Audio Device)" <{0.0.1.00000000}.{678523a2-9932-4c08-bb38-0d232f7fab01}>, "Внутренний разъем  AUX (High Definition Audio Device)" <{0.0.1.00000000}.{7192fed5-4886-4660-8672-62467a55dd6a}>, "Внутренний разъем  AUX (AMD High Definition Audio Device)" <{0.0.1.00000000}.{bd1e3f8a-5c62-4883-acf4-57149707e857}>, "Line In (High Definition Audio Device)" <{0.0.1.00000000}.{c3c7911c-d159-4b57-bcbb-101f18f68811}>, "Microphone (High Definition Audio Device)" <{0.0.1.00000000}.{dba3decc-6cfd-4103-93c9-516bc183f08a}> }, full list refresh: false
[2024.10.24 14:39:22] Media Info: Type Camera, default: OBS Virtual Camera, list: { "OBS Virtual Camera" <OBS Virtual Camera> }, full list refresh: true
[2024.10.24 14:39:22] App Info: reading settings...
[2024.10.24 14:39:22] App Info: reading encrypted settings...
[2024.10.24 14:39:22] Lang Info: Loaded cached, keys: 7530
[2024.10.24 14:39:22] Font: from ':/gui/fonts/OpenSans-Regular.ttf' loaded 'Open Sans'
[2024.10.24 14:39:22] Font: from ':/gui/fonts/OpenSans-Italic.ttf' loaded 'Open Sans'
[2024.10.24 14:39:22] Font: from ':/gui/fonts/OpenSans-SemiBold.ttf' loaded 'Open Sans Semibold'
[2024.10.24 14:39:22] Font: from ':/gui/fonts/OpenSans-SemiBoldItalic.ttf' loaded 'Open Sans Semibold'
[2024.10.24 14:39:22] Font: from ':/gui/fonts/Vazirmatn-UI-NL-Regular.ttf' loaded 'Vazirmatn UI NL'
[2024.10.24 14:39:22] Font: from ':/gui/fonts/Vazirmatn-UI-NL-SemiBold.ttf' loaded 'Vazirmatn UI NL SemiBold'
[2024.10.24 14:39:22] OpenAL Logging Level: (not set)
[2024.10.24 14:39:22] App Info: reading accounts info...
[2024.10.24 14:39:22] App Info: reading encrypted info...
[2024.10.24 14:39:22] App Info: reading map...
[2024.10.24 14:39:22] App Info: reading encrypted map...
[2024.10.24 14:39:22] App Info: reading encrypted user settings...
[2024.10.24 14:39:22] App Info: encrypted user settings read.
[2024.10.24 14:39:22] App Info: reading encrypted mtp data...
[2024.10.24 14:39:22] MTP Info: read keys, current: 5, to destroy: 0
[2024.10.24 14:39:22] Map read time: 5
[2024.10.24 14:39:22] App Info: reading encrypted mtp config...
[2024.10.24 14:39:22] Export Info: Destroy top bar by controller removal.
[2024.10.24 14:39:22] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2024.10.24 14:39:23] OpenGL Profile: OpenGLES.
[2024.10.24 14:39:23] OpenGL Renderer: ANGLE (AMD, AMD Radeon(TM) Graphics Direct3D11 vs_5_0 ps_5_0, D3D11-31.0.21910.5)
[2024.10.24 14:39:23] OpenGL Vendor: Google Inc. (AMD)
[2024.10.24 14:39:23] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.10 git hash: e3f59e8d0c3e)
[2024.10.24 14:39:23] OpenGL Extensions: GL_EXT_conservative_depth, GL_EXT_map_buffer_range, GL_OVR_multiview2, GL_KHR_debug, GL_EXT_debug_label, GL_OES_sample_variables, GL_EXT_base_instance, GL_OES_texture_half_float, GL_EXT_polygon_offset_clamp, GL_OES_compressed_ETC2_sRGB8_texture, GL_NV_framebuffer_blit, GL_EXT_blend_func_extended, GL_OES_texture_npot, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_OES_compressed_EAC_RG11_signed_texture, GL_ANGLE_get_tex_level_parameter, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_ANGLE_get_serialized_context_string, GL_OES_draw_elements_base_vertex, GL_EXT_texture_rg, GL_EXT_read_format_bgra, GL_EXT_texture_format_BGRA8888, GL_EXT_color_buffer_float, GL_EXT_texture_storage, GL_ANGLE_pack_reverse_row_order, GL_ANGLE_request_extension, GL_CHROMIUM_bind_uniform_location, GL_EXT_frag_depth, GL_NV_pixel_buffer_object, GL_EXT_texture_mirror_clamp_to_edge, GL_ANGLE_texture_compression_dxt3, GL_EXT_clip_cull_distance, GL_ANGLE_lossy_etc_decode, GL_CHROMIUM_bind_generates_resource, GL_EXT_shader_texture_lod, GL_ANGLE_texture_usage, GL_EXT_discard_framebuffer, GL_ANGLE_program_cache_control, GL_OES_EGL_image, GL_ANGLE_shader_pixel_local_storage, , GL_EXT_multi_draw_indirect, GL_APPLE_clip_distance, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_texture_stencil8, GL_ANGLE_clip_cull_distance, GL_EXT_unpack_subimage, GL_OES_texture_border_clamp, GL_ANGLE_robust_client_memory, GL_OES_texture_half_float_linear, GL_NV_pack_subimage, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_ANGLE_multiview_multisample, GL_ANGLE_base_vertex_base_instance, GL_EXT_texture_compression_rgtc, GL_OES_compressed_EAC_R11_signed_texture, GL_ANGLE_provoking_vertex, GL_OES_EGL_image_external, GL_EXT_debug_marker, GL_CHROMIUM_copy_compressed_texture, GL_EXT_EGL_image_external_wrap_modes, GL_OES_fbo_render_mipmap, GL_ANGLE_copy_texture_3d, GL_EXT_robustness, GL_ANGLE_framebuffer_multisample, GL_ANGLE_multi_draw, GL_OES_compressed_EAC_R11_unsigned_texture, GL_CHROMIUM_lose_context, GL_ANGLE_translated_shader_source, GL_EXT_texture_compression_bptc, GL_EXT_texture_compression_s3tc_srgb, GL_OES_rgb8_rgba8, GL_OES_mapbuffer, GL_OES_standard_derivatives, GL_OES_texture_float, GL_CHROMIUM_copy_texture, GL_NV_shader_noperspective_interpolation, GL_CHROMIUM_sync_query, GL_EXT_texture_type_2_10_10_10_REV, GL_KHR_parallel_shader_compile, GL_NV_fence, GL_OES_draw_buffers_indexed, GL_OES_surfaceless_context, GL_EXT_draw_elements_base_vertex, GL_EXT_sRGB, GL_OES_texture_float_linear, GL_EXT_texture_border_clamp, GL_NV_EGL_stream_consumer_external, GL_ANGLE_framebuffer_blit, GL_ANGLE_instanced_arrays, GL_EXT_texture_compression_dxt1, GL_ANGLE_renderability_validation, GL_EXT_instanced_arrays, GL_OES_compressed_ETC2_RGB8_texture, GL_ANGLE_polygon_mode, GL_EXT_disjoint_timer_query, GL_EXT_texture_filter_anisotropic, GL_EXT_blend_minmax, GL_OES_compressed_ETC2_RGBA8_texture, GL_OES_element_index_uint, GL_ANGLE_stencil_texturing, GL_EXT_draw_buffers, GL_OES_shader_multisample_interpolation, GL_ANGLE_memory_size, GL_EXT_float_blend, GL_EXT_occlusion_query_boolean, GL_EXT_depth_clamp, GL_ANGLE_base_vertex_base_instance_shader_builtin, GL_EXT_render_snorm, GL_WEBGL_video_texture, GL_OES_depth32, GL_OES_packed_depth_stencil, GL_EXT_multisampled_render_to_texture, GL_ANGLE_client_arrays, GL_ANGLE_shader_pixel_local_storage_coherent, GL_EXT_color_buffer_half_float, GL_EXT_texture_norm16, GL_OES_vertex_array_object, GL_OES_get_program_binary, GL_EXT_draw_buffers_indexed, GL_OVR_multiview, GL_AMD_performance_monitor, GL_ANGLE_texture_multisample, GL_ANGLE_texture_compression_dxt5, GL_EXT_clip_control, GL_OES_depth24, GL_ANGLE_depth_texture, GL_OES_EGL_image_external_essl3
[2024.10.24 14:39:23] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_no_config_context, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2024.10.24 14:39:23] OpenGL: QOpenGLContext created, version: 3.3.
[2024.10.24 14:39:23] OpenGL: [TRUE] (Window)
[2024.10.24 14:39:24] RPC Error: request 63 got fail with code 400, error CHANNEL_PRIVATE
[2024.10.24 14:39:39] RPC Error: request 80 got fail with code 400, error PEER_ID_INVALID
[2024.10.24 14:41:00] Assertion Failed! "ptr_ != nullptr" pointers:98
```

**Comment by Aokromes** [comment]
confirmed 
Crash ID: 9eb25f96-a125-4fa5-9f57-dcdd93af5206

**Comment by Aokromes** [comment]
fixed on 5.6.4
